examples/anonymous/index.html

```diff
@@ -49,16 +49,30 @@
   <body>
     <div id="firechat-wrapper"></div>
     <script type="text/javascript">
-      var chatRef = new Firebase("https://firechat-demo.firebaseio.com");
+      // Initialize Firebase SDK
+      // TODO: use firechat-demo config; ask @raynor for it
```

Comment:
TODO: update config to the firechat-demo config.
_(Mirroring comment from previous PR)_